### PR TITLE
net-im/dino: Does not working calls button

### PR DESCRIPTION
**Pull request by KoNekoD**
Could you see what can be done with this functionality?
It's hard without calls😞

### CODE REVIEW

File: net-im/dino/dino-0.4.2.ebuild [hunk 1]
```diff
@@ -58,6 +58,7 @@ src_configure() {
 	vala_setup
 
 	# TODO: Make videocalls (rtp) optional and not completely disable it
+	# This
```
[the remark this answers is not in the document]
Comment (KoNekoD) [reply net-im/dino/dino-0.4.2.ebuild:61]: I asked the official dino repository why there are no calls on gentoo, they said the problem is that our maintainer specifically disabled it

**Comment by KoNekoD** [comment]
How do you do that? I couldn't find any instructions

File: net-im/dino/dino-0.4.2.ebuild (hunk 1 above)
[the remark this answers is not in the document]
Comment (thesamesam) [reply net-im/dino/dino-0.4.2.ebuild:61]: Right, so let's handle this on Bugzilla (bugs.gentoo.org).

**Comment by KoNekoD** [comment]
https://bugs.gentoo.org/903179
I created it

**Comment by KoNekoD** [comment]
@thesamesam

**Comment by jubalh** [comment]
I responded on bugzilla. Let's further discuss there.

**Pull request closed without merging**